### PR TITLE
fix byte consumer/producer issue

### PR DESCRIPTION
**Pull request by emlin**
to make it parse error correctly

Together with #6012, fixes: #5819

### CODE REVIEW

File: lib/apiservers/portlayer/restapi/configure_port_layer.go [hunk 1]
```diff
@@ -17,6 +17,9 @@ package restapi
 import (
 	"context"
 	"crypto/tls"
+	"encoding/json"
+	errs "errors"
```
Comment (caglar10ur) [inline lib/apiservers/portlayer/restapi/configure_port_layer.go]: fmt.Errorf and you no longer need to import this :)

**Comment by morris-jason** [comment]
Should we make the same change for TarConsumer/Producer?

File: lib/apiservers/portlayer/restapi/configure_port_layer.go (hunk 1 above)
Comment (willsherwood) [inline lib/apiservers/portlayer/restapi/configure_port_layer.go]: Inconsistent with how we have imported "errors" without the alias in all other files.

File: lib/apiservers/portlayer/restapi/configure_port_layer.go [hunk 2]
```diff
@@ -163,3 +167,38 @@ func setupMiddlewares(handler http.Handler) http.Handler {
 func setupGlobalMiddleware(handler http.Handler) http.Handler {
 	return handler
 }
+
```
Comment (willsherwood) [inline lib/apiservers/portlayer/restapi/configure_port_layer.go]: exported function might need a brief comment to explain what it does, like ByteStreamProducer.

**Comment by emlin** [comment]
> Should we make the same change for TarConsumer/Producer?

This is actually a good question. Did you ever find this kind of error? I could not reproduce it even I broken the cp process.

File: lib/apiservers/portlayer/restapi/configure_port_layer.go (hunk 1 above)
Comment (emlin) [inline lib/apiservers/portlayer/restapi/configure_port_layer.go]: this is from @casualjim 😄

**Pull request merged**